frontend/src/User/pages/About/About.jsx

```
@@ -0,0 +1,171 @@
+      <style>
+        {`
+          @import url('https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap');
+
+          body {
+            font-family: 'Press Start 2P', cursive;
+          }
+        `}
+      </style>
+
+      {/* Hero Section */}
+      <section ref={el => (sectionRefs.current[0] = el)} className="relative py-20 px-4">
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)